chart/values.yaml

```diff
@@ -587,6 +587,34 @@ loki-stack:
               - __meta_kubernetes_pod_uid
               - __meta_kubernetes_pod_container_name
               target_label: __path__
+
+nats:
```

Comment:
Needs comment stating what this nats is for...

Reply:
Should we have an `eventing` component which in turn enables or disables NATS ?

Reply:
The options under nats are those of nats subchart, so we cant rename this.